### PR TITLE
New: Update Visual Studio Marketplace listing

### PR DESCRIPTION
**Pull request by antross**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

## Short description of the change(s)

Clarify this is a preview, fix the name, and categorize as a linter.

### CODE REVIEW

File: packages/extension-vscode/package.json [hunk 1]
```diff
@@ -46,6 +50,7 @@
   "license": "Apache-2.0",
   "main": "./dist/src/extension",
   "name": "vscode-webhint",
+  "preview": true,
```
Comment (alrra) [inline packages/extension-vscode/package.json]: You already have this, below.

Comment (alrra) [reply packages/extension-vscode/package.json]: never mind..

**Comment by alrra** [comment]
@antross In the future for patch version use `Fix:`. Thanks!

**Comment by alrra** [comment]
@antross Did (will) publish a new version?

**Comment by antross** [comment]
> @antross Did (will) publish a new version?
@alrra yes, the update is already live in the marketplace.

**Comment by alrra** [comment]
> the update is already live in the marketplace.

👍

**Pull request closed without merging**